README.md

```diff
@@ -108,7 +108,7 @@ Tento řádek obsahuje **tučné písmo**, *kurzívu*, a `monospace`
 
 ## Převod souboru v rozšíření Markdown
 
-Na převod textových souboru lze použit speciálně nástroje, jako například následující:
+Ke konverzi textových souborů lze použít k tomu určené nástroje, jako například:
```

Comment:
@lruzicka takže nakonec konverze sedí lépe nez převod?

Reply:
Konverze = převod. To je fakt. Pro soubory to dokonce sedí víc než "převod", ale špatně se ti to vyslovovalo. V každém případě "na převod textových" souborů není správně česky, ale "k převodu (či ke konverzi) lze použít. To byl můj point. 
To vzniklo tak, že původně tam bylo "na konverzi" a to jsem opravil :crocodile:

Reply:
Dekuji za opraveni, hotovo